### PR TITLE
Factor out some test logic

### PR DESCRIPTION
**Pull request by tbantle22**
Mario's feedback from https://github.com/dolthub/hosted-cypress/pull/50

### CODE REVIEW

File: cypress/e2e/publicPaths/render/contact/index.spec.ts [hunk 1]
```diff
@@ -1,41 +1,23 @@
...
-    newExpectation(
-      "should have Support Ticket link",
-      "[data-cy=support-ticket]",
-      beVisible,
-    ),
+    shouldFindAndContain("[data-cy=contact-us-page]", [
+      "Contact Us",
+      "We usually respond within 24 hours",
+    ]),
+    shouldBeVisible("[data-cy=join-discord]", "Discord button"),
```
Comment (ElTragon) [inline cypress/e2e/publicPaths/render/contact/index.spec.ts:17]: heads up this is why the tests are failing, it should be `join-discord`
Same for below

**Review by ElTragon — APPROVED** [review]
lgtm

nit, would it be good to change our funcs so they be called like this?
`shouldBeVisible("[data-cy=terms-page] header", "Header")`
Might be worth it but I don't know if this would be going a bit to far.

File: cypress/e2e/publicPaths/render/contact/index.spec.ts (hunk 1 above)
Comment (tbantle22) [reply cypress/e2e/publicPaths/render/contact/index.spec.ts:17]: Oops, thanks

**Comment by tbantle22** [comment]
> lgtm
> 
> nit, would it be good to change our funcs so they be called like this? `shouldBeVisible("[data-cy=terms-page] header", "Header")` Might be worth it but I don't know if this would be going a bit to far.

I feel like that might be going too far. I think using just `data-cy` is more common and is a more helpful abstraction

**Pull request merged**